### ISSUE TITLE
Add prop to terra-application-layout to control terra-application-utility button aria label

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-framework#831]
# Feature Request

## Description
Currently, the utility button in application-layout defaults to the following aria-label: `aria-label="Utility Button"`

UX has identified a need to customize this to improve accessibility. Planning to add a prop to allow teams to pass in their own text for the aria-label for this button.

**Comment by bjankord** [comment]
@scottwilmarth Can we get some guidance on verbiage for this. It would be nice if we could handle this programmatically based on the display text of the button.

**Comment by scottwilmarth** [comment]
@bjankord I spoke with Justin Eddings, and we come to a more meaningful name. Instead of "Utility Button", we think it should be "User Options". It is more descriptive and reflective of the content is opens.
